Makefile

```diff
@@ -17,7 +17,8 @@ build:
 
 .PHONY: test
 test:
-	$(MAKE) -C pkg/policy-compiler test
+	$(MAKE) -C connectors test
+	$(MAKE) -C pkg test
```

Comment:
I suggest to remove this (and the makefile in pkg) from this PR. It will break tests unless ugly workarounds are added.

Reply:
Related to my other comment about this running recursively also the vault tests.